src/System.Collections.Sequences/System/Collections/Sequences/Position.cs

```diff
@@ -12,15 +11,31 @@ public struct Position : IEquatable<Position>
         long _index;
         object _item;
 
+        public static Position Create<T>(T item, long index) where T : class
+        {
+            Position position = default;
+            position.Set(item, index);
+            return position;
+        }
+
         public static explicit operator int(Position position) => (int)position._index;
 
         public static implicit operator long(Position position) => position._index;
 
-        public long GetIndex() => _index;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is it necessary to mark all these APIs with AggresiveInlining attribute? I would think the JIT would automatically inline them.

As per Andy, anything under 16 bytes of IL gets inlined.

cc @AndyAyersMS, @vancem

Reply:
Yeah, I will actually remove all the ones that are one liners.